addons/crm/wizard/crm_module_access_request.py

```diff
@@ -0,0 +1,10 @@
+from odoo import models
+
+
+class CrmModuleAccessRequest(models.TransientModel):
```

Comment:
Not sure what it does, but this should be removed. It adds a layer on top of an existing feature that does everything you want. Just cleanly set attributes / parameters when calling the send request method to give the template. But adding a model adds stuff in registry, environment, ... we should avoid models when possible, and here it does not really add something.